notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb

```diff
@@ -430,43 +436,6 @@
     "print(f\"FP32 F1: {fp32_f1:.3f}\")"
    ]
   },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "NNCF configuration can be defined in a json file or a dictionary. See the [NNCF quantization documentation](https://github.com/openvinotoolkit/nncf/blob/develop/docs/compression_algorithms/Quantization.md) for more information on the possible values."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# NNCF uses the model loaded at the beginning of this notebook. If after quantizing the model, you\n",
-    "# want to quantize with a different config, reload the model by uncommenting the next two lines\n",
-    "#\n",
-    "# model = monai.networks.nets.BasicUNet(spatial_dims=2, in_channels=1, out_channels=1).eval()\n",
-    "# model.load_state_dict(new_state_dict)\n",
-    "\n",
-    "nncf_config_dict = {\n",
-    "    \"input_info\": {\"sample_size\": [1, 1, 512, 512]},\n",
-    "    \"target_device\": \"CPU\",\n",
-    "    \"compression\": {\n",
-    "        \"algorithm\": \"quantization\",\n",
-    "        # performance preset uses symmetric weights and activations\n",
-    "        \"preset\": \"performance\",\n",
-    "        # Do not quantize LeakyReLU activations to allow the INT8 model to run on Intel GPU\n",
-    "        \"ignored_scopes\": [\"{re}.*LeakyReLU*\"],\n",
```

Comment:
Are you sure that configuration is not actual anymore?
 At least ignored scope, could you please test int8 model inference on iGPU?

Reply:
You are right, it doesn't work on iGPU. I returned this configuration settings.